sphinx_automodapi/tests/helpers.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import os
+import sys
+from copy import deepcopy
+
+from .test_cases import write_conf, SPHINX_LT_17, build_main
```

Comment:
`SPHINX_LT_17` and similar are defined in each files separately. If we move this into one central place then please do it for the rest of them, too.

Reply:
Will do

Reply:
Fixed